<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticketDefine</class>
 <widget class="QWidget" name="ticketDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Accept_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tickets:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
